images/bay-php/settings.php

```diff
@@ -174,8 +174,8 @@
 // Aggregate JavaScript files on
 $config['system.performance']['js']['preprocess'] = 1;
 
-if ($smtp_on = getenv('ENABLE_SMTP')) {
-  $config['smtp.settings']['smtp_on'] = (strtolower($smtp_on) == "true");
```

Comment:
We incorrectly assumed that this variable controlled whether the SMTP module was used.